src/icons/icons.tsx

```diff
@@ -1,9 +1,15 @@
+export {
+  BiChevronDown as ChevronDown,
+  BiChevronRight as ChevronRight
```

Comment:
use the Md version of chevron `import { MdChevronRight } from "react-icons/md";` left and right for constistency?